### PR TITLE
Updated HTTPS rules for TheAtlantic.com

### PR DESCRIPTION
**Pull request by joshmaker**
The Atlantic (www.theatlantic.com) now supports HTTPS. As a result the falsemixed rule file The_Atlantic.com-falsemixed.xml has been deleted and a redirect from http://www.theatlantic.com to https://www.theatlantic.com has been added to The_Atlantic.com.xml

I ran the tests I could, but was unable to run the full test suite on my Macbook because `./install-dev-dependencies.sh` fails for me locally (Mac OS 10.11.6, Homebrew 0.9.9):

```
$ ./install-dev-dependencies.sh
+ set -o errexit -o xtrace
+ '[' 1365886654 '!=' 0 ']'
+ SUDO_SHIM=sudo
+ type apt-get
./install-dev-dependencies.sh: line 10: type: apt-get: not found
+ type brew
+ brew list python
+ brew install libxml2 gnu-sed chromedriver
Warning: libxml2-2.9.4 already installed
Warning: gnu-sed-4.2.2 already installed
==> Downloading https://chromedriver.storage.googleapis.com/2.23/chromedriver_mac32.zip

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "chromedriver"
Download failed: https://chromedriver.storage.googleapis.com/2.23/chromedriver_mac32.zip
```

Happy to do further testing if possible.

### CODE REVIEW

File: src/chrome/content/rules/The_Atlantic.com.xml [hunk 1]
```diff
@@ -11,70 +9,23 @@
...
-
 -->
-<ruleset name="The Atlantic.com (partial)">
+<ruleset name="The Atlantic.com">
 
 	<!--	Direct rewrites:
 				-->
+	<target host="www.theatlantic.com" />
 	<target host="cdn.theatlantic.com" />
 	<target host="ssl.theatlantic.com" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/The_Atlantic.com.xml]: Please note as `Expired` in a [top comment](https://github.com/EFForg/https-everywhere/search?utf8=%E2%9C%93&q=problematic)

File: src/chrome/content/rules/The_Atlantic.com.xml [hunk 2]
```diff
@@ -11,70 +9,23 @@
...
-	³ Rule disabled by default
-	⁴ Rule disabled by default <= mismatched cert
-	⁵ Unsecurable <= 503, akamai
-
 -->
-<ruleset name="The Atlantic.com (partial)">
+<ruleset name="The Atlantic.com">
 
 	<!--	Direct rewrites:
 				-->
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/The_Atlantic.com.xml]: Please also add `<target host="theatlantic.com" />`

File: src/chrome/content/rules/The_Atlantic.com.xml [hunk 3]
```diff
@@ -11,70 +9,23 @@
...
 	<target host="ssl.theatlantic.com" />
 
 	<!--	Complications:
 				-->
 	<target host="assets.theatlantic.com" />
 
-
 	<!--	Not secured by server:
 					-->
 	<!--securecookie host="^\.theatlantic\.com$" name="^(SC_LINKS|atl_tests|s_cc|s_sq)$" /-->
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/The_Atlantic.com.xml]: Could this be `<securecookie host=".+" name=".+" />` ?

File: src/chrome/content/rules/The_Atlantic.com.xml (hunk 1 above)
Comment (joshmaker) [inline src/chrome/content/rules/The_Atlantic.com.xml]: We've disable the ssl.theatlantic.com domain name so it is no longer accessible. Do I need to note that in a top comment?

File: src/chrome/content/rules/The_Atlantic.com.xml (hunk 2 above)
Comment (joshmaker) [inline src/chrome/content/rules/The_Atlantic.com.xml]: Done

File: src/chrome/content/rules/The_Atlantic.com.xml (hunk 1 above)
Comment (gloomy-ghost) [inline src/chrome/content/rules/The_Atlantic.com.xml]: Don't worry about it if it is not existing anymore.

**Comment by gloomy-ghost** [comment]
If you are the owner of this site and it's full https covered, I will suggest you to consider [HSTS Preload](https://hstspreload.appspot.com)

File: src/chrome/content/rules/The_Atlantic.com.xml (hunk 3 above)
Comment (joshmaker) [inline src/chrome/content/rules/The_Atlantic.com.xml]: I'm looking into the cookies now, would it be possible to get the other updates merged in while I confirm that this won't cause any bugs for us?

**Comment by joshmaker** [comment]
> If you are the owner of this site and it's full https covered, I will suggest you to consider HSTS Preload

For SEO and testing purposes, we're staggering the tranisiton of our users to the HTTPS version of the site. I'll definitely add 'HSTS Preload' as a final step in our transition plan.

**Comment by joshmaker** [comment]
Just wanted to check in -- I'm not sure what still needs to be done on my end to get this merged in

**Comment by gloomy-ghost** [comment]
I'm not a official reviewer here, so I was just pointing something I had been asked to change.
As you can see there are hundreds of pending PRs. Please be patient to wait the reviewers.

**Comment by gloomy-ghost** [comment]
Thanks, merged.

**Pull request merged**